### DISCUSSION TITLE
unknown option "-justify"

### DISCUSSION THREAD

**Discussion by brianlin725** [TomSchimansky/CustomTkinter#312] [Q&A] [answered]
My code was working well before I did a pip upgrade.
Not sure if this is CustomTkinter itself but I thought to come here since it only happened after the pip upgrade
![image](https://user-images.githubusercontent.com/67332442/180720308-21628f03-6fb7-46ec-ab55-2f62fdb1751b.png)

Are there other options to make my entry text align to the right?

**Comment by TomSchimansky** [comment, marked as answer]
You should use .configure instead of .config. Looks like I removed .config from CTkEntry, maybe I will add it again. .config is not implemented on most CTk widgets, so you should always use .configure.
